Src/Sarif.PatternMatcher.Sdk/StaticValidatorBase.cs

```diff
@@ -20,16 +53,14 @@ public IEnumerable<ValidationResult> IsValidStatic(IDictionary<string, FlexMatch
                     continue;
                 }
 
-                string scanTarget = groups["scanTargetFullPath"].Value;
-                string key = $"{scanTarget}#{validationResult.Fingerprint}";
```

Comment:
>{scanTarget}# [](http://example.com/codeflow?start=31&length=13)

Per description, this is a perfile fingerprint hash.
With that, I would expect to see this in the new changes as well.

Otherwise, we have a fingerprintCache only.